website/docs/tutorial/1c-setting-up-databricks.md

```diff
@@ -5,10 +5,244 @@ id: setting-up-databricks
+
+4. Once that's done, let’s ensure that we can query the training data that we loaded earlier.  Navigate back to the SQL space through the left hand menu and choose `SQL Editor`.  This will bring you to a query editor.
+
+5. Ensure that you can run a select * from each of the tables with the following code snippets. 
+
+```sql
+select * from default.jaffle_shop_customers
+```
+
+```sql
+select * from default.jaffle_shop_orders
+```
+
+```sql
+select * from stripe.payments
+```
+
+<Lightbox src="/img/databricks_tutorial/images/query_check.png" title="Query Check" />
+
+6. To get setup in dbt Cloud, you will now need to get the SQL Endpoint connection information and generate a user token. 
+
+You can find your SQL endpoint connection information by going to the `Databricks UI > SQL > SQL Endpoints > Starter Endpoint > Connection details`. Save this information or keep this window open for the next section.
+
+<Lightbox src="/img/databricks_tutorial/images/SQL_Endpoint_Details.png" title="Databrick SQL Endpoint Connection Information" />
+
+7. To generate a user token for your development credentials in dbt Cloud, click on `Settings` on the left side console (while still in the SQL part of the workspace). Click on `Personal Access Token` and provide a comment like `dbt Cloud development`. Save the token information somewhere because you will need it for the next part. 
+
+<Lightbox src="/img/databricks_tutorial/images/Generate_user_token.png" title="Generate User Token" />
+
+
+Congratulations! At this point, you have created a Databricks account, loaded training data, and successfully set up a SQL end point to query the data.  
+
 ## Set up a project to connect dbt Cloud to Databricks
+
+Now it’s time to connect to dbt Cloud to Databricks in order to develop your dbt Project. 
+
+If you don't have an existing dbt Cloud account, you can [sign up for a dbt Cloud trial account here](https://www.getdbt.com/signup/). If you already have a dbt Cloud account, you can create a new project in your existing account.  In this tutorial, we will be using the Set Up project workflow for new users.  This can easily be adapted for additional projects to an existing account.
+
+1. Log into dbt Cloud in a new window. Once you have logged into your new account and validated your email, you will see our Project Setup Flow with the page Set up “Analytics”.  Click `Continue`. 
+
+<Lightbox src="/img/databricks_tutorial/images/setup_flow_dbt_cloud.png" title="dbt Cloud Set Up Flow" />
+
+2. Choose `Databricks` on the Set Up a Database Connection page. 
+
+<Lightbox src="/img/databricks_tutorial/images/setup_database_connection.png" title="Databrick Connection" />
+
+3. Configure your connection to Databricks this way: 
+
+    a. For the name, write `Databricks` or another simple title.
+    
+    b. For Databricks settings, reference your SQL endpoint connection details from part 6 of the previous section for each of the following fields:
```

Comment:
```suggestion
    b. For Databricks settings, reference your SQL endpoint connection details from step 6 of the previous section for each of the following fields:
```
quick tweak: changing to step instead of part